words/core.asm

```diff
@@ -5426,6 +5426,8 @@ z_sign:         rts
         ; """
 
 xt_slash:
+                jsr underflow_2
```

Comment:
both of these are flagged in headers as having UF so the `1 strip-underflow !` benchmarks were broken